package.json

```diff
@@ -4,7 +4,8 @@
   "description": "JSON Web Token implementation (symmetric and asymmetric)",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --require test/util/fakeDate && nsp check && cost-of-modules"
+    "lint": "eslint .",
+    "test": "mocha --require test/util/fakeDate && npm run lint && nsp check && cost-of-modules"
```

Comment:
In #486 It was mentioned to break the build early when linting fails and it was therefor proposed to move the lint in front of the test.

Reply:
I've moved linting first.